src/main/java/org/jabref/gui/preview/PreviewViewer.java

```diff
@@ -112,6 +112,13 @@ public PreviewViewer(BibDatabaseContext database, DialogService dialogService, S
 
     }
 
+    public void setTheme(String theme){
+        if(theme.equals("Dark.css")){
+            previewView.getEngine().setUserStyleSheetLocation(ClipBoardManager.class.getResource("Dark.css").toString());
```

Comment:
What's the reason to use `Clipboardmanager` here instead of `JabRefFrame`. The latter is a bit more canonical and a better base point.

Reply:
@tobiasdiez  I assumed since there was no JabRefFrame referenced at all in the file, instead of importing it and then using it. It would be better to use the class that is already being used and is in same directory as JabRefFrame.

However, I understand your point, and will make changes right away!